src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -61,117 +62,91 @@
        (max 1)
        (min 4)))
 
-(schema/defn ^:always-validate
```

Comment:
I don't think you should need `default-pool-size` above this anymore (there's one in jruby-utils).